features/named-color.yml

```diff
@@ -0,0 +1,7 @@
+name: Named colors
+description: The `named-color` CSS values are a standard set of colors that can be referenced by a name, such as `red` or `limegreen`.
+spec: https://drafts.csswg.org/css-color-4/#named-colors
+group: css
+compat_features:
+  - css.types.color.named-color
+  - css.types.color.named-color.rebeccapurple
```

Comment:
It won't change the baseline high status, but using: `compute_from: css.types.color.named-color` would make it seem like named colors have always been supported (earliest versions of browsers), which they were (one color was added later, but probably fine to hide the date details).